specialMenu.php

```diff
@@ -15,17 +15,17 @@
 		<div class="accordion-item">
 			<h2 class="accordion-header" id="monday">
 				<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseOne" aria-expanded="false" aria-controls="flush-collapseOne">
-					Milk Chocolate Mondays
+					Milk Chocolate Monday
```

Comment:
If we make one singular we should make them all singular

Reply:
makes sense and im on it!